docs/Platforms.md

```diff
@@ -76,15 +76,44 @@ A marketing site by its nature will have fewer moving parts than a full-fledged
 
 We build mobile apps using native technologies, meaning that they interface directly with the libraries provided by the operating system, that being iOS and Android.
 
-We do not recommend mobile apps built in a web browser. The level of fit and finish is not the same as you get from native. We do however use cross-platform frameworks where appropriate.
+We do not recommend mobile apps built in a web browser. The level of fit and finish is not the same as you get from native. We do however use cross-platform frameworks where appropriate. For example, the following are some cross-platform libraries for the basic necessities of any mobile application:
 
-### Native Apps
+* [Fabric](https://get.fabric.io/) - Crash reporting
+* [Amplitude](https://amplitude.com/) - App metrics/events
+* [Realm](https://realm.io/) - Persistent local storage/database
```

Comment:
I think it'd be great to add a paragraph on the pros/cons of those. Fabric and Amplitude especially are big dependencies (file-size and perhaps even performance). Is there a scenario when it doesn't make sense to include those?